Fastpack/GraphBuilder.ml

```diff
@@ -183,23 +183,26 @@ let read_module
         Lwt.catch
           (fun () -> process location source)
           (function
-           | FlowParser.Parse_error.Error args ->
-             let location_str = Module.location_to_string location in
-             Lwt.fail (Context.PackError (ctx, CannotParseFile (location_str, args)))
-           | Preprocessor.Error message ->
-             Lwt.fail (Context.PackError (ctx, PreprocessorError message))
-           | FastpackUtil.Error.UnhandledCondition message ->
-             Lwt.fail (Context.PackError (ctx, UnhandledCondition message))
-           | exn ->
-             Lwt.fail exn
+            | FlowParser.Parse_error.Error args ->
+              let location_str = Module.location_to_string location in
+              let src = match source with 
```

Comment:
In what situation do we not have the `source` for a file?

Reply:
The only case when source is None is when you have an `import/require` statement like this:
```javascript
import './some-loader!'
```
Quite rare case imho, but still possible.